src/models/users.model.js

```diff
@@ -0,0 +1,53 @@
+const { AppDataSource } = require("./data-source");
+
+
+const createUser = async (name, email, hashedPw, phoneNumber) => {
+  await AppDataSource.query(`
+  INSERT INTO users (                    
+  pw,
+  email, 
+  name,
+  phone_number
+  )
+  VALUES (
+  '${hashedPw}', 
+  '${email}',
+  '${name}',
+  '${phoneNumber}'
+  )
+`);
+};
+
+
+async function getUserByEmail(email) {
+  const users = await AppDataSource.query(`
+    SELECT
+      email,
+      pw AS password
+    FROM
+      users
+    WHERE
+      email = "${email}"
+    ;
+  `);
+  return users[0];
+}
+
```

Comment:
```suggestion

const getUserByEmail = async(email) => {
  const [user] = await AppDataSource.query(`
    SELECT
      email,
      pw AS password
    FROM
      users
    WHERE
      email = "${email}"
    ;
  `);
  return user;
}

```

1. 위와 마찬가지로 화살표 함수를 활용해 주세요.
2. [user] 로 이용하시면, 함수 호출 결과가 배열일 때, 첫번째 요소를 곧바로 users 변수에 할당할 수 있습니다.
3. 더불어, 유저가 한 명 뿐이므로 `users`라는 복수 단어는 데이터 의미에 혼선을 줄 수 있습니다 :) user로 변경하는 것을 추천드립니다.